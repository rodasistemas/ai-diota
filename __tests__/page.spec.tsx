import renderer from 'react-test-renderer';
import Home from '@/app/page';
import { JSX } from 'react';

describe('<Home>', () => {
  it('should expose a function', () => {
		expect(Home).toBeDefined();
	});
  const renderTree = (tree: JSX.Element) => renderer.create(tree);

  it('page should return expected output', () => {
    expect(renderTree(<Home />).toJSON()).toMatchSnapshot();
  });
});