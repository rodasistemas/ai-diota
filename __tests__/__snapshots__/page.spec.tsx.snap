// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Home> page should return expected output 1`] = `
<div>
  page
</div>
`;
